
Header header
string type
string classe
string color
int64 x
int64 y
int64 w
int64 h